app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt

```diff
@@ -308,12 +308,38 @@ class ContactPatientBottomSheet : BaseBottomSheet<
     setAppointmentReminderView.enableNextReminderDateStepper()
   }
 
-  override fun showCallResult(callResult: Outcome, updatedAt: Instant) {
-    // Nothing to look at here, yet.
+  override fun showCallResult() {
+    callPatientView.callResultOutcomeViewVisible = true
   }
 
   override fun hideCallResult() {
-    // Nothing to look at here, yet.
+    callPatientView.callResultOutcomeViewVisible = false
+  }
+
+  override fun setupAgreedToVisitCallResultOutcome() {
+    callPatientView.callResultOutcomeText = getString(R.string.call_result_outcome_agreed_to_visit)
+    callPatientView.setupCallResultViewForAgreedToVisit()
+  }
+
+  override fun setupRemindToCallLaterCallResultOutcome(daysToRemindAppointmentIn: Int) {
+    val dayOrDays = if (daysToRemindAppointmentIn == 1) {
+      getString(R.string.call_result_outcome_remind_to_call_later_day)
+    } else {
+      getString(R.string.call_result_outcome_remind_to_call_later_days)
+    }
```

Comment:
FYI, we can use `plurals` strings for this. Take a look at `patientsummary_contact_doctor_patient_info_bps` in `strings.xml`